scripts/z3950_view.py

```diff
@@ -130,7 +130,7 @@ def GET(self, ia):
         if from_marc_xml:
             ret += '<h2>From MARC binary on archive.org</h2>'
 
-            ret += 'record length: ' + `len(meta_mrc)` + ' bytes<br>'
+            ret += 'record length: ' + repr(len(meta_mrc)) + ' bytes<br>'
```

Comment:
It seems many of these `repr()` are unnecessary. it's not clear what they are adding `repr(<int>)`?

Reply:
Yes, just len(meta_mrc). If there's some weird edge case where this doesn't work str(len())

Reply:
I wasn't sure of this as well! What should be done here?

Reply:
@hornc can you probably tell where all are the `repr` unnecessary. I'll replace all of them in a different PR.